Postgres and crossview pods failed to start

I am trying to install crossviewin my local kind cluster, using helm option:
helm install crossview crossview/crossview \  
  --namespace crossview \
  --create-namespace \
  --set secrets.dbPassword=secret \
  --set secrets.sessionSecret=$(openssl rand -base64 32)
Postgres and crossview pods failed to start, I noticed this error in postgres logs: FATAL: role "-d" does not exist

**postgres log:**
chmod: /var/run/postgresql: Operation not permitted
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/crossview ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default time zone ... UTC
creating configuration files ... ok
running bootstrap script ... ok
sh: locale: not found
2025-12-17 18:12:57.198 UTC [29] WARNING:  no usable system locales were found
performing post-bootstrap initialization ... ok
syncing data to disk ... ok
initdb: warning: enabling "trust" authentication for local connections
initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/crossview -l logfile start

waiting for server to start....2025-12-17 18:12:57.584 UTC [36] LOG:  starting PostgreSQL 16.3 on aarch64-unknown-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
2025-12-17 18:12:57.585 UTC [36] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-12-17 18:12:57.586 UTC [39] LOG:  database system was shut down at 2025-12-17 18:12:57 UTC
2025-12-17 18:12:57.588 UTC [36] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-12-17 18:12:57.702 UTC [36] LOG:  received fast shutdown request
2025-12-17 18:12:57.704 UTC [36] LOG:  aborting any active transactions
2025-12-17 18:12:57.705 UTC [36] LOG:  background worker "logical replication launcher" (PID 42) exited with exit code 1
2025-12-17 18:12:57.705 UTC [37] LOG:  shutting down
2025-12-17 18:12:57.705 UTC [37] LOG:  checkpoint starting: shutdown immediate
2025-12-17 18:12:57.726 UTC [37] LOG:  checkpoint complete: wrote 924 buffers (5.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.006 s, sync=0.014 s, total=0.022 s; sync files=301, longest=0.005 s, average=0.001 s; distance=4267 kB, estimate=4267 kB; lsn=0/191AA60, redo lsn=0/191AA60
2025-12-17 18:12:57.728 UTC [36] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-12-17 18:12:57.812 UTC [1] LOG:  starting PostgreSQL 16.3 on aarch64-unknown-linux-musl, compiled by gcc (Alpine 13.2.1_git20240309) 13.2.1 20240309, 64-bit
2025-12-17 18:12:57.812 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-12-17 18:12:57.812 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-12-17 18:12:57.813 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-12-17 18:12:57.815 UTC [52] LOG:  database system was shut down at 2025-12-17 18:12:57 UTC
2025-12-17 18:12:57.817 UTC [1] LOG:  database system is ready to accept connections
2025-12-17 18:13:03.432 UTC [62] FATAL:  role "-d" does not exist
2025-12-17 18:13:05.248 UTC [67] ERROR:  duplicate key value violates unique constraint "pg_class_relname_nsp_index"
2025-12-17 18:13:05.248 UTC [67] DETAIL:  Key (relname, relnamespace)=(users_id_seq, 2200) already exists.
2025-12-17 18:13:05.248 UTC [67] STATEMENT:  CREATE TABLE "users" ("id" bigserial,"username" text NOT NULL,"email" text NOT NULL,"password_hash" text NOT NULL,"role" text DEFAULT 'user',"first_name" text,"last_name" text,"created_at" timestamptz,"updated_at" timestamptz,PRIMARY KEY ("id"))
2025-12-17 18:13:08.441 UTC [77] FATAL:  role "-d" does not exist
2025-12-17 18:13:13.429 UTC [84] FATAL:  role "-d" does not exist
2025-12-17 18:13:18.445 UTC [91] FATAL:  role "-d" does not exist
2025-12-17 18:13:23.438 UTC [98] FATAL:  role "-d" does not exist
2025-12-17 18:13:26.760 UTC [105] FATAL:  role "-d" does not exist
2025-12-17 18:13:28.423 UTC [114] FATAL:  role "-d" does not exist
2025-12-17 18:13:33.432 UTC [122] FATAL:  role "-d" does not exist
2025-12-17 18:13:36.757 UTC [129] FATAL:  role "-d" does not exist
2025-12-17 18:13:38.443 UTC [136] FATAL:  role "-d" does not exist
2025-12-17 18:13:43.441 UTC [143] FATAL:  role "-d" does not exist


**crossview pod logs:**

Defaulted container "crossview" out of: crossview, wait-for-db (init)
2025-12-17T18:14:01.465Z	PANIC	commands/serve.go:41	Failed to run database migrations: insufficient arguments
crossview-go-server/commands.(*ServeCommand).Run.func1
	/app/commands/serve.go:41
reflect.Value.call
	/usr/local/go/src/reflect/value.go:581
reflect.Value.Call
	/usr/local/go/src/reflect/value.go:365
go.uber.org/dig.defaultInvoker
	/go/pkg/mod/go.uber.org/dig@v1.17.0/container.go:238
go.uber.org/dig.(*Scope).Invoke
	/go/pkg/mod/go.uber.org/dig@v1.17.0/invoke.go:160
go.uber.org/fx.runInvoke
	/go/pkg/mod/go.uber.org/fx@v1.17.1/invoke.go:93
go.uber.org/fx.(*module).executeInvoke
	/go/pkg/mod/go.uber.org/fx@v1.17.1/module.go:174
go.uber.org/fx.(*module).executeInvokes
	/go/pkg/mod/go.uber.org/fx@v1.17.1/module.go:155
go.uber.org/fx.New
	/go/pkg/mod/go.uber.org/fx@v1.17.1/app.go:534
crossview-go-server/commands.WrapSubCommand.func1
	/app/commands/commands.go:38
github.com/spf13/cobra.(*Command).execute
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920
github.com/spf13/cobra.(*Command).ExecuteC
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044
github.com/spf13/cobra.(*Command).Execute
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
main.main
	/app/main.go:15
runtime.main
	/usr/local/go/src/runtime/proc.go:285
panic: Failed to run database migrations: insufficient arguments

goroutine 1 [running]:
go.uber.org/zap/zapcore.CheckWriteAction.OnWrite(0x0?, 0x0?, {0x0?, 0x0?, 0x0?})
	/go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:198 +0x74
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0x40005001a0, {0x0, 0x0, 0x0})
	/go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:264 +0x1a0
go.uber.org/zap.(*SugaredLogger).log(0x40005ff688, 0x4, {0x1d63a0b?, 0x1c?}, {0x40000b2d48?, 0x1?, 0x1?}, {0x0, 0x0, 0x0})
	/go/pkg/mod/go.uber.org/zap@v1.24.0/sugar.go:295 +0xe0
go.uber.org/zap.(*SugaredLogger).Panicf(...)
	/go/pkg/mod/go.uber.org/zap@v1.24.0/sugar.go:189
crossview-go-server/commands.(*ServeCommand).Run.func1({0x400052a5a0, 0x2, 0x40000b2eb8?}, {{0x4000010045, 0x4}, {0x4000056021, 0xa}, {0x0, 0x0}, {0x4000010028, ...}, ...}, ...)
	/app/commands/serve.go:41 +0x1e4
reflect.Value.call({0x19de340?, 0x1e515a8?, 0x0?}, {0x1d254d1, 0x4}, {0x40000e6120, 0x6, 0x8?})
	/usr/local/go/src/reflect/value.go:581 +0x960
reflect.Value.Call({0x19de340?, 0x1e515a8?, 0x8?}, {0x40000e6120?, 0x40001f7601?, 0x40000e6120?})
	/usr/local/go/src/reflect/value.go:365 +0x94
go.uber.org/dig.defaultInvoker({0x19de340?, 0x1e515a8?, 0x40001f7620?}, {0x40000e6120?, 0x6?, 0x2095b68?})
	/go/pkg/mod/go.uber.org/dig@v1.17.0/container.go:238 +0x28
go.uber.org/dig.(*Scope).Invoke(0x40002d7b80, {0x19de340, 0x1e515a8}, {0x0, 0x0, 0x40000b38e8?})
	/go/pkg/mod/go.uber.org/dig@v1.17.0/invoke.go:160 +0x450
go.uber.org/fx.runInvoke({0x206ab80?, 0x40002d7b80?}, {{0x19de340, 0x1e515a8}, {0x40005de120, 0x6, 0x7}})
	/go/pkg/mod/go.uber.org/fx@v1.17.1/invoke.go:93 +0x104
go.uber.org/fx.(*module).executeInvoke(0x400015e990, {{0x19de340, 0x1e515a8}, {0x40005de120, 0x6, 0x7}})
	/go/pkg/mod/go.uber.org/fx@v1.17.1/module.go:174 +0xdc
go.uber.org/fx.(*module).executeInvokes(0x400015e990)
	/go/pkg/mod/go.uber.org/fx@v1.17.1/module.go:155 +0x74
go.uber.org/fx.New({0x40000b3cb0, 0x2, 0x2?})
	/go/pkg/mod/go.uber.org/fx@v1.17.1/app.go:534 +0x648
crossview-go-server/commands.WrapSubCommand.func1(0x40000cad00?, {0x1d25421?, 0x4?, 0x1d25425?})
	/app/commands/commands.go:38 +0x200
github.com/spf13/cobra.(*Command).execute(0x4000209208, {0x4000126010, 0x0, 0x0})
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x5c8
github.com/spf13/cobra.(*Command).ExecuteC(0x33b3420)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
main.main()
	/app/main.go:15 +0x24

--- a/helm/crossview/templates/service.yaml
+++ b/helm/crossview/templates/service.yaml
@@ -13,7 +13,7 @@ spec:
   type: {{ .Values.service.type }}
   ports:
     - port: {{ .Values.service.port }}
-      targetPort: http
+      targetPort: {{ .Values.service.targetPort }}
       protocol: TCP
       name: http
   selector:

--- a/src/presentation/components/layout/Layout.jsx
+++ b/src/presentation/components/layout/Layout.jsx
@@ -46,11 +46,11 @@ export const Layout = ({ children }) => {
   }, [isInClusterMode]);
 
   const bgColor = getBackgroundColor(colorMode, 'html');
-  const totalLeftWidth = sidebarWidth + contextSidebarWidth;
+  const totalLeftWidth = sidebarWidth + (isInClusterMode ? 0 : contextSidebarWidth);
 
   return (
     <Box minH="100vh" bg={bgColor}>
-      {showContextSidebar && <ContextSidebar />}
+      {!isInClusterMode && showContextSidebar && <ContextSidebar />}
       <Sidebar onToggle={handleSidebarToggle} onResize={handleSidebarResize} />
       <Box 
         ml={`${totalLeftWidth}px`} 

--- a/src/presentation/components/layout/Sidebar.jsx
+++ b/src/presentation/components/layout/Sidebar.jsx
@@ -28,6 +28,10 @@ export const Sidebar = ({ onToggle, onResize }) => {
   const { user, logout, kubernetesRepository, selectedContext, colorMode, selectedContextError, isInClusterMode } = useAppContext();
 
   useEffect(() => {
+    if (isInClusterMode) {
+      setContextSidebarWidth(0);
+      return;
+    }
     const updateContextSidebarWidth = () => {
       const saved = localStorage.getItem('contextSidebarCollapsed');
       setContextSidebarWidth(saved === 'true' ? 0 : 60);
@@ -38,7 +42,7 @@ export const Sidebar = ({ onToggle, onResize }) => {
     };
     window.addEventListener('contextSidebarWidthChanged', handleWidthChange);
     return () => window.removeEventListener('contextSidebarWidthChanged', handleWidthChange);
-  }, []);
+  }, [isInClusterMode]);
 
   useEffect(() => {
     const currentWidth = isCollapsed ? 60 : width;
@@ -224,7 +228,7 @@ export const Sidebar = ({ onToggle, onResize }) => {
       }}
       transition={isResizing ? 'none' : 'width 0.2s, left 0.2s'}
       position="fixed"
-      left={`${contextSidebarWidth}px`}
+      left={`${isInClusterMode ? 0 : contextSidebarWidth}px`}
       top={0}
       zIndex={1000}
       display="flex"
@@ -274,7 +278,7 @@ export const Sidebar = ({ onToggle, onResize }) => {
               </Text>
               </HStack>
             )}
-            {!isCollapsed && (
+            {!isCollapsed && !isInClusterMode && (
               <Box
                 as="button"
                 onClick={() => {

